src/globemap.ts

```diff
@@ -1292,18 +1291,13 @@ module powerbi.extensibility.visual {
             const controlElements: Element = this.createControlElements();
             controlContainer.appendChild(controlElements);
             this.root.append(controlContainer);
-            function onMouseDown(callback: (element: SVGElement) => void) {
-                (d3.event as MouseEvent).stopPropagation();
-                if ((d3.event as MouseEvent).button === 0) {
-                    callback(<SVGElement> (d3.event as MouseEvent).currentTarget);
-                }
-            }
-            this.zoomContainer = d3.select(controlContainer);
-            this.zoomContainer
-                .selectAll("g")
-                .on("mousedown", () => onMouseDown(
-                    (element: SVGElement) => {
-                        const controlType = element.classList.toString().split(" ").filter(className => className.search("js-") !== -1)[0];
+            this.zoomContainer = document.querySelector("div.controls-container");
```

Comment:
zoomContainer is actually controlContainer in the previous version

Reply:
Can we use controlContainer instead?

Reply:
Fixed